BigCommerce/UserInterface/01a_Connections.md

```diff
@@ -0,0 +1,198 @@
+
+- *Name*   
+    Connection name. 
+
+- ![Edit](../../Assets/Icons/Edit02.png "[Edit]") (Edit)  
+    Click this button to edit the connection name.
+
+- ![Apply](../../Assets/Icons/Check.png "[Apply]") (Apply)  
+    Click this button to apply the changes to the connection name.  This button is only displayed if you are editing the connection name.
+
+- *Created DD/MM/YYYY*  
+    Creation date of the connection. This field is read-only.  
+
+
+
+## Edit BigCommerce connection &ndash; Credentials
+
+*Omni-Channel > Settings > Tab CONNECTIONS > Select BigCommerce connection > Tab Credentials*
+
+![Edit BigCommerce credentials](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/EditConnectionCredentials.png "[Edit BigCommerce credentials]")
+
+- *Driver*  
+    Driver name. For *BigCommerce*, the *BigCommerce* driver is displayed. This drop-down list is read-only.
+
+**Credentials** 
+
+- *Store hash*   
+    Click the field to edit the hash key of the *BigCommerce* store.
+
+- *Access token*   
+    Click the field to edit the access token for the *BigCommerce* connection.
+
+- [SAVE]  
+    Click this button to save any changes made.
+
+
+
+## Edit BigCommerce connection &ndash; Settings
+
+*Omni-Channel > Settings > Tab CONNECTIONS > Select BigCommerce connection > Tab Settings*
+
+![Edit BigCommerce settings](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/EditConnectionSettings.png "[Edit BigCommerce settings]")
+
+The *Settings* tab is composed of the following setting entries:
+- [Variant option type](#variant-option-type)
+- [Order filters](#order-filters)
+
+
+### Variant option type
+
+*Omni-Channel > Settings > Tab CONNECTIONS > Select BigCommerce connection > Tab Settings > Menu entry Variant option type*
+
+![Variant option type](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/VariantOptionType.png "[Variant option type]")
+
+- *Variant option type used for variants*   
+   Use this dropdown list to select the type to select variants. For detailed information, see [Configure BigCommerce connection](../Integration/01_ManageBigCommerceConnection.md#configure-bigcommerce-connection).
+
+
+
+### Order filters
+
+*Omni-Channel > Settings > Tab CONNECTIONS > Select BigCommerce connection > Tab Settings > Menu entry Order filters*
+
+![Order filters](../../Assets/Screenshots/Channels/Settings/Connections/BigCommerce/OrderFilters.png "[Order filters]")
+
+- *Filter orders to import by*   
+   Enable the toggles at the order status to define the orders to be imported to *Actindo*.  
+   For detailed information on the *BigCommerce* order statuses see the *BigCommerce* documentation under [https://developer.bigcommerce.com/docs/rest-management/orders/order-status#get-all-order-statuses](https://developer.bigcommerce.com/docs/rest-management/orders/order-status#get-all-order-statuses).
+   
+- *Import only orders with an ID greater than*   
```

Comment:
In Integration-Datei als "Import only orders with an ID as of"... Bitte vereinheitlichen. (As of finde ich auch schöner, aber evtl. würde ich hier beim "greater than" bleiben, da es nicht falsch ist, und das ist, was im UI steht. Vielleicht kriegen wir die translation files mal...). ; )

Reply:
Wollte Jannik noch ändern